### DISCUSSION TITLE
Not listed in the .NET Foundation list of projects

### DISCUSSION THREAD

**Discussion by GeorgeLeithead** [CommunityToolkit/Maui#583] [General]
I was looking at the .NET Foundation list of projects, and this .NET MAUI Community Toolkit is not listed, but the Xamarin.Forms Community Toolkit is.  Should it be?

[.NET Foundation - Projects](https://dotnetfoundation.org/projects)

**Comment by TheCodeTraveler** [comment]
Good catch George! I've pinged the .NET Foundation team to figure out how we can add them to the list on the website!

**Reply by GeorgeLeithead** [reply]
Honeslty, just a fluke of circumstance. I only noticed beause I couldn't find the YT channel for last nights stand-up, and realised it was in a different channel ".NET Foundation" compared to the "dotNET" channel (subscription added BTW) .  Then today, I happened to see a tweet announcing the elimination of the $100 membership fee [Tweet](https://twitter.com/dotnetfdn/status/1565697302202908678?s=20&t=rHEFuQktWsdUq4w7XMFdYw), and thought I'd check out their web site, as never done so before!

**Reply by GeorgeLeithead** [reply]
Also note that they are aware that their SSL certificate expired 3 days ago.

**Reply by GeorgeLeithead** [reply]
> Also note that they are aware that their SSL certificate expired 3 days ago.

The dot net Foundation appears to have resolved their SSL certificate issue.

**Comment by CliffAgius** [comment]
Closed as answered
